From 5ad16bf39ffd0d8d977f53d89edf40ef1cc32e40 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:26:37 +0800
Subject: [PATCH] Revert "android: logger: Add new system log for
 framework/system log messages"

This reverts commit ae991396e9c66ca7bb73fceb62e632ff4ed6c4e0.
---
 drivers/staging/android/logger.c | 7 -------
 drivers/staging/android/logger.h | 1 -
 2 files changed, 8 deletions(-)

diff --git a/drivers/staging/android/logger.c b/drivers/staging/android/logger.c
index 15b2e97002aa..64cc2a1e7b16 100644
--- a/drivers/staging/android/logger.c
+++ b/drivers/staging/android/logger.c
@@ -557,7 +557,6 @@ static struct logger_log VAR = { \
 DEFINE_LOGGER_DEVICE(log_main, LOGGER_LOG_MAIN, 64*1024)
 DEFINE_LOGGER_DEVICE(log_events, LOGGER_LOG_EVENTS, 256*1024)
 DEFINE_LOGGER_DEVICE(log_radio, LOGGER_LOG_RADIO, 64*1024)
-DEFINE_LOGGER_DEVICE(log_system, LOGGER_LOG_SYSTEM, 64*1024)
 
 static struct logger_log *get_log_from_minor(int minor)
 {
@@ -567,8 +566,6 @@ static struct logger_log *get_log_from_minor(int minor)
 		return &log_events;
 	if (log_radio.misc.minor == minor)
 		return &log_radio;
-	if (log_system.misc.minor == minor)
-		return &log_system;
 	return NULL;
 }
 
@@ -605,10 +602,6 @@ static int __init logger_init(void)
 	if (unlikely(ret))
 		goto out;
 
-	ret = init_log(&log_system);
-	if (unlikely(ret))
-		goto out;
-
 out:
 	return ret;
 }
diff --git a/drivers/staging/android/logger.h b/drivers/staging/android/logger.h
index 2cb06e9d8f98..a562434d7419 100644
--- a/drivers/staging/android/logger.h
+++ b/drivers/staging/android/logger.h
@@ -32,7 +32,6 @@ struct logger_entry {
 
 #define LOGGER_LOG_RADIO	"log_radio"	/* radio-related messages */
 #define LOGGER_LOG_EVENTS	"log_events"	/* system/hardware events */
-#define LOGGER_LOG_SYSTEM	"log_system"	/* system/framework messages */
 #define LOGGER_LOG_MAIN		"log_main"	/* everything else */
 
 #define LOGGER_ENTRY_MAX_LEN		(4*1024)
-- 
2.35.3

